src/Log.php

```diff
@@ -97,6 +93,14 @@ public function __construct(
         $this->fullText = mb_convert_encoding($text, 'UTF-8', 'UTF-8');
     }
 
+    protected function getLogContentPattern($level): string
```

Comment:
Suggestion: Add type on $level parameter

Reply:
Didn't need this function after updating from `main`. Thanks